### PR TITLE
Update validation-codes.md with errors E069 to E091.

### PR DESCRIPTION
**Pull request by julianmorley**
Closes out https://github.com/OCFL/spec/issues/446

### CODE REVIEW

File: validation/validation-codes.md [hunk 1]
```diff
@@ -78,6 +78,29 @@
...
+| E069 | 'An OCFL Storage Root MUST contain a Root Conformance Declaration identifying it as such.' | https://ocfl.io/draft/spec/#E069
+| E070 | 'If present, [the ocfl_layout.json document] MUST include include the following two keys in the root JSON object: [key, description]' | https://ocfl.io/draft/spec/#E070
+| E071 | 'The value of [the ocfl_layout.json key "key"] is not defined in the OCFL specification, but MUST correspond to a value given in an Extension.' | https://ocfl.io/draft/spec/#E071
+| E072 | 'Sub-directories within an OCFL Storage Root MUST NOT contain files that are not part of an OCFL Object.' | https://ocfl.io/draft/spec/#E072
+| E073 | 'Empty directories MUST NOT appear within a storage root.' | https://ocfl.io/draft/spec/#E073
+| E074 | 'Although implementations may require multiple OCFL Storage Roots - that is, several logical or physical volumes, or multiple "buckets" in an object store - each OCFL Storage Root MUST be independent.' | https://ocfl.io/draft/spec/#E074
+| E075 | 'The OCFL version declaration <span id="E075">MUST</span> be formatted according to the NAMASTE specification.' | https://ocfl.io/draft/spec/#E075
```
Comment (zimeon) [inline validation/validation-codes.md:87]: Remove \<span>

File: validation/validation-codes.md [hunk 2]
```diff
@@ -78,6 +78,29 @@
...
+| E084 | 'Storage hierarchies MUST NOT include files within intermediate directories.' | https://ocfl.io/draft/spec/#E084
+| E085 | 'Storage hierarchies MUST be terminated by OCFL Object Roots.' | https://ocfl.io/draft/spec/#E085
+| E086 | 'The storage root extensions directory MUST conform to the same guidelines and limitations as those defined for object extensions.' | https://ocfl.io/draft/spec/#E086
+| E087 | 'An OCFL validator MUST ignore any files in the storage root it does not understand.' | https://ocfl.io/draft/spec/#E087
+| E088 | 'Additional files MUST NOT appear in other directories under the storage root.' | https://ocfl.io/draft/spec/#E088
+| E089 | 'If the preservation of non-OCFL-compliant features is required then the content MUST be wrapped in a suitable disk or filesystem image format which OCFL can treat as a regular file.' | https://ocfl.io/draft/spec/#E089
+| E090 | 'Hard and soft (symbolic) links are not portable and <span id="E090">MUST NOT</span> be used within OCFL Storage hierachies.' | https://ocfl.io/draft/spec/#E090
```
Comment (zimeon) [inline validation/validation-codes.md:102]: Remove \<span>

File: validation/validation-codes.md [hunk 3]
```diff
@@ -78,6 +78,29 @@
 | E066 | 'Each version block in each prior inventory file must represent the same object state as the corresponding version block in the current inventory file.' | https://ocfl.io/draft/spec/#E066
 | E067 | 'The extensions directory must not contain any files, and no sub-directories other than extension sub-directories.' | https://ocfl.io/draft/spec/#E067
 | E068 | 'The specific structure and function of the extension, as well as a declaration of the registered extension name must be defined in one of the following locations: The OCFL Extensions repository OR The Storage Root, as a plain text document directly in the Storage Root.' | https://ocfl.io/draft/spec/#E068
+| E069 | 'An OCFL Storage Root MUST contain a Root Conformance Declaration identifying it as such.' | https://ocfl.io/draft/spec/#E069
+| E070 | 'If present, [the ocfl_layout.json document] MUST include include the following two keys in the root JSON object: [key, description]' | https://ocfl.io/draft/spec/#E070
```
Comment (awoods) [inline validation/validation-codes.md:82]: Typo: "include include"

File: validation/validation-codes.md [hunk 4]
```diff
@@ -78,6 +78,29 @@
...
+| E084 | 'Storage hierarchies MUST NOT include files within intermediate directories.' | https://ocfl.io/draft/spec/#E084
+| E085 | 'Storage hierarchies MUST be terminated by OCFL Object Roots.' | https://ocfl.io/draft/spec/#E085
+| E086 | 'The storage root extensions directory MUST conform to the same guidelines and limitations as those defined for object extensions.' | https://ocfl.io/draft/spec/#E086
+| E087 | 'An OCFL validator MUST ignore any files in the storage root it does not understand.' | https://ocfl.io/draft/spec/#E087
+| E088 | 'Additional files MUST NOT appear in other directories under the storage root.' | https://ocfl.io/draft/spec/#E088
+| E089 | 'If the preservation of non-OCFL-compliant features is required then the content MUST be wrapped in a suitable disk or filesystem image format which OCFL can treat as a regular file.' | https://ocfl.io/draft/spec/#E089
+| E090 | 'Hard and soft (symbolic) links are not portable and <span id="E090">MUST NOT</span> be used within OCFL Storage hierachies.' | https://ocfl.io/draft/spec/#E090
```
Comment (awoods) [inline validation/validation-codes.md:102]: There still appears to one `span` remaining.

Comment (zimeon) [reply validation/validation-codes.md:102]: Darn, I didn't spot that :-(

**Comment by ahankinson** [comment]
Sorry @zimeon I didn’t mean to dismiss your review.

File: validation/validation-codes.md (hunk 4 above)
Comment (zimeon) [reply validation/validation-codes.md]: I just fixed it, reduces the pool of approvers but we should have enough folks to merge after the weekend I hope!

**Comment by zimeon** [comment]
You all can decide whether my :+1: counts ;-)

**Pull request merged**